### DISCUSSION TITLE
Question about string field mutation

### DISCUSSION THREAD

**Discussion by BEGEMOT9I** [react-hook-form/react-hook-form#8468] [Q&A]
Hi! I have come across a strange behavior where slashes in a text field are escaped (you can test this with a basic example - https://codesandbox.io/s/react-hook-form-basic-6mlii, add a slash to the text field and hit submit). Is there any way to control this behavior (in my case i don't need an escaping at all)?

**Comment by bluebill1049** [comment]
build a controlled component?

**Reply by BEGEMOT9I** [reply]
Yes, but the question is rather whether there is an out-of-the-box api for this and why does this happen at all?

**Reply by BEGEMOT9I** [reply]
@bluebill1049 Hey! Sorry to bother you, but is there still an option to disable this behavior without creating a custom control?
